test/package-check-success.yml

```diff
@@ -0,0 +1,13 @@
+- tasks:
+  - name: install ansible
+    yum: name=ansible state=2.1.0.0
+
+  - name: install ansible-lint
+    pip: name=ansible-lint
+    args:
+      state: 3.1.2
```

Comment:
this wouldn't work in reality - it should be

```
  pip: name=ansible-lint-3.1.2
```

state can only usually be `present` (typically default) or `latest` (pip has more options but it's latest that is bad)